### PR TITLE
completed dom-event-delegation exercise - Colin Macrae

### PR DESCRIPTION
**Pull request by colin-macrae**

### CODE REVIEW

File: dom-event-delegation/main.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+var taskListItem = document.querySelector('.task-list-item');
+
+taskList.addEventListener('click', taskListCallback);
+function taskListCallback(e) {
+  console.log('event target:', e.target);
+  console.log('event tagname:', e.target.tagName);
+  if (e.target && e.target.nodeName === 'BUTTON') {
+
+    console.log(taskList.closest('.task-list-item'));
+    taskListItem.remove();
```
Comment (shawnkost) [inline dom-event-delegation/main.js:11]: You don't want to remove `taskListItem`, you want to remove the `event.target.closest('.task-list-item')`

**Review by shawnkost — CHANGES_REQUESTED** [review]
One change here Colin!

File: dom-event-delegation/main.js (hunk 1 above)
Comment (colin-macrae) [reply dom-event-delegation/main.js:11]: I was looking at that and I understand it now.  Will go back and review it also.  Thanks!

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**